cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -44,6 +44,14 @@
      */
     public static class Push extends ReplicatorBuilder<Datastore, URI, Push> {
 
+        private int changeLimitPerBatch = 500;
+
+        private int batchLimitPerRun = 100;
+
+        private int bulkInsertSize = 10;
+
```

Comment:
Any thought about putting these three values that seem to be reused throughout into a constants class?